build/ci-scripts/run-api-tests.sh

```diff
@@ -0,0 +1,17 @@
+cd build/ci-scripts/
+cp api-testing.config.json ../../.api-testing.config.json
+mkdir log
+docker-compose up -d
+docker-compose logs -f --no-color > "log/wikibase.api-testing.log" &
```

Comment:
Do these logs end up anywhere where we can see them?

Reply:
yeah if you go to this https://github.com/wmde/WikibaseReconcileEdit/actions/runs/857711510  for example and scroll down you'll see TestArtifacts